build-system/babel-config/minified-config.js

```diff
@@ -60,7 +60,9 @@ function getMinifiedConfig(buildFor = 'preact') {
       './build-system/babel-plugins/babel-plugin-amp-mode-transformer',
       BUILD_CONSTANTS,
     ],
-    ['@babel/plugin-transform-for-of', {loose: true, allowArrayLike: true}],
+    !(argv.esm || argv.sxg)
```

Comment:
nit: do we use this condition in more places? if yes, could be nice to create a local var `
```js
const isNomodule = !(argv.esm || argv.sxg)
```